copy-all.sls

```diff
@@ -5,7 +5,20 @@ SPDX-FileCopyrightText: 2024 seven-beep <ebn@entreparentheses.xyz>
 SPDX-License-Identifier: AGPL-3.0-or-later
 #}
 
-{%- if salt['pillar.get']('qusal:dotfiles:all', default=true) == true -%}
+{%- if (
+         ( salt['pillar.get']('qusal:dotfiles:all', default=true) == true  )
+           or salt['pillar.get']('qusal:dotfiles:dom0') == true
+           or salt['pillar.get']('qusal:dotfiles:git')  == true
+           or salt['pillar.get']('qusal:dotfiles:gtk')  == true
+           or salt['pillar.get']('qusal:dotfiles:net')  == true
+           or salt['pillar.get']('qusal:dotfiles:pgp')  == true
+           or salt['pillar.get']('qusal:dotfiles:sh')   == true
+           or salt['pillar.get']('qusal:dotfiles:ssh')  == true
+           or salt['pillar.get']('qusal:dotfiles:vim')  == true
+           or salt['pillar.get']('qusal:dotfiles:x11')  == true
+           or salt['pillar.get']('qusal:dotfiles:xfce') == true
```

Comment:
Aren't all these lines starting with `or` missing the `, default=true`?

Reply:
@seven-beep response by mail: https://github.com/ben-grande/dotfiles/pull/1#issuecomment-2403349384

> No, the logic here is someone may set qusal:dotfiles:all to false and individually activate, eg: qusal:dotfiles:dom0 to true, allowing then a whitelist or a blacklist.
>
> This is admissibly a bit contrived and while I think about it, to make it works properly, I should also edit the jinja of the other states in the same spirit.
>
> Do you think this behavior is desirable ?

Yes, I believe that if `qusal:dotfiles:all` is set to `false`, all dotfiles must be skipped.